### ISSUE TITLE
Stop treating all tag values with OSM Wiki page as recommended

### ISSUE THREAD

**Issue by matkoniecz** [openstreetmap/iD#5460]
tl;dr: [OSM Wiki page describing highway=unsurfaced as an outdated tagging scheme](https://wiki.openstreetmap.org/wiki/Tag:highway%3Dunsurfaced) is causing iD to propose this tag to mappers. This is undesirable.

![highway unsurfaced in id](https://user-images.githubusercontent.com/899988/47951415-b1f0c180-df60-11e8-9dfe-bdc9db5e04ea.png)

-----

There are some tags that were used in past but nowadays are almost unused and should not be used ever again. Part of handling this situation is to create page on OSM wiki describing why tag is problematic and to mention possible alternatives.

It leads to one unfortunate side effect. As I undestand, in raw tag editing mode, iD proposes tags that are either with some minimal usage or have their own OSM wiki page.

https://github.com/openstreetmap/iD/blob/master/modules/services/taginfo.js#L93

So tags like

https://wiki.openstreetmap.org/wiki/Tag:noexit%3Dno
https://wiki.openstreetmap.org/wiki/Tag:natural%3Dland
https://wiki.openstreetmap.org/wiki/Tag:highway%3Dunsurfaced
https://wiki.openstreetmap.org/wiki/Tag:landuse%3Dwood
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Deducation
https://wiki.openstreetmap.org/wiki/Tag:highway%3Dabandoned
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dproposed
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Ddisused
https://wiki.openstreetmap.org/wiki/Tag:highway%3Dbyway
https://wiki.openstreetmap.org/wiki/Tag:amenity%3Dhotel
https://wiki.openstreetmap.org/wiki/Tag:natural%3Dmoor

and other will be proposed by iD for use even after their usage reaches 0.

Is there any situation where almost unused tag should be promoted just because it has its own OSM wiki page? I think that only completely failed proposals will fail to get `>0.0%` usage, and promoting them is not necessary.

I propose to stop considering "has wiki page" as sufficient reason to suggest tags and rely just on Taginfo usage.

It is related to #4508 that proposed much stronger and more controversial change.

**Comment by matkoniecz** [comment]
I suspect that changing

> return parseFloat(d.fraction) > 0.0 || d.in_wiki;

to 

> return parseFloat(d.fraction) > 0.0;

is sufficient to avoid this problem without causing unwanted side effects.

**Comment by matkoniecz** [comment]
PR opened at https://github.com/openstreetmap/iD/pull/5461
